src/Core/Models/Api/Public/Request/MemberCreateRequestModel.cs

```diff
@@ -22,7 +23,8 @@ public override OrganizationUser ToOrganizationUser(OrganizationUser existingUse
 
         public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
         {
-            if (Email.Contains(" ") || Email.Contains("<"))
+            var strictEmailAttribute = new StrictEmailAddressAttribute();
```

Comment:
This shouldn't be required here... you should be able to simply remove the additional email validation here all-together since the `[StrictEmailAddress]` attribute will be validated _before_ `Validate(validationContext)` is called (similar to the `[Required]` attributes)

Reply:
Good point, now that I look at this again I don't know why we're doing validation here rather than just using attributes. I'll refactor it to use attributes instead and give it a test to make sure I haven't broken anything.

Reply:
This is now done. The error message strings have changed slightly but the structure of the response is otherwise identical, and this brings it into line with our other validation checks, so I see no reason to keep the `Validate` method.